context.jsonld

```diff
@@ -1,30 +1,31 @@
 {
```

Comment:
This is an automatic file, you do not have to make amendments here. if you need a customized IRI you can use the notes_context.jsonld file (see [contribution manual](https://bit.ly/contribution_manual))

Reply:
got ya!